principia.props

```diff
@@ -0,0 +1,195 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--Define project configurations.  Nothing interesting can happen here.-->
+  <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="Debug|Win32">
+      <Configuration>Debug</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Debug|x64">
+      <Configuration>Debug</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release_LLVM|Win32">
+      <Configuration>Release_LLVM</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release_LLVM|x64">
+      <Configuration>Release_LLVM</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|Win32">
+      <Configuration>Release</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|x64">
+      <Configuration>Release</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+  </ItemGroup>
+  
+  <!--Microsoft C++ stuff.-->
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
+
+  <!--Definitions.-->
+  <PropertyGroup>
+    <!--Predicates.-->
+    <PrincipiaCompilerClangLLVM>false</PrincipiaCompilerClangLLVM>
+    <PrincipiaCompilerClangLLVM Condition="$(Configuration)==Release_LLVM">true</PrincipiaCompilerClangLLVM>
+    <PrincipiaOptimize>false</PrincipiaOptimize>
+    <PrincipiaOptimize Condition="$(Configuration)==Release or
+                                  $(Configuration)==Release_LLVM">true</PrincipiaOptimize>
+    <PrincipiaTestProject>true</PrincipiaTestProject>
+    <PrincipiaTestProject  Condition="$(ProjectName) == ksp_plugin or
+                                      $(ProjectName) == serialization or
+                                      $(ProjectName) == benchmarks or
+                                      $(ProjectName) == tools or
+                                      $(ProjectName) == mathematica">false</PrincipiaTestProject>
+    <!--Dependency paths.-->
+    <PrincipiaDependencyConfiguration>Debug</PrincipiaDependencyConfiguration>
+    <PrincipiaDependencyConfiguration Condition="$(PrincipiaOptimize)">Release</PrincipiaDependencyConfiguration>
```

Comment:
This feels vaguely circular with the definition of `PrincipiaOptimize`, above.

Reply:
Yes but no because Release_LLVM. We don't want to require that glog be built with clang when we try to get a better error message on quantities.